build/dependencies.props

```diff
@@ -236,7 +236,7 @@
     <SystemIdentityModelTokensJwtPackageVersion>5.3.0</SystemIdentityModelTokensJwtPackageVersion>
     <SystemNetHttpPackageVersion>4.3.2</SystemNetHttpPackageVersion>
     <SystemReactiveLinqPackageVersion>3.1.1</SystemReactiveLinqPackageVersion>
-    <SystemReflectionEmitPackageVersion>4.3.0</SystemReflectionEmitPackageVersion>
+    <SystemReflectionEmitPackageVersion>4.6.0-preview.18606.1</SystemReflectionEmitPackageVersion>
```

Comment:
Is that something that needs to be added to https://github.com/dotnet/versions/pull/409 ?

Reply:
This looks right to me.

Reply:
Can you move this version up near the other files that updated (near the comment "Packages from dotnet/corefx ")?